playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerView.java

```diff
@@ -437,5 +448,40 @@ public void setSurfaceAspectRatioResizeMode(PKAspectRatioResizeMode resizeMode)
         }
         return exoPlayerAspectRatioResizeMode;
     }
+
+    /**
+     * Creates new cue configuration if `isIgnoreCueSettings` is set to true by application
+     * Checks if the application wants to ignore the in-stream CueSettings otherwise goes with existing Cue configuration
+     *
+     * @param cueList cue list coming in stream
+     * @param subtitleViewPosition subtitle view position configuration set by application
+     * @return List of modified Cues
+     */
+    public List<Cue> getModifiedSubtitlePosition(List<Cue> cueList, PKSubtitlePosition subtitleViewPosition) {
+        if (!cueList.isEmpty()) {
```

Comment:
@GouravSna  please check if cueList could be null

Reply:
Done